django/otto/templates/components/user_cost.html

```diff
@@ -0,0 +1,22 @@
+<div id="cost-bar-outer"
+     hx-get="{% url 'user_cost' %}"
+     hx-swap="outerHTML"
+     hx-trigger='load delay:60000ms'
+     data-bs-toggle="tooltip"
+     data-bs-placement="bottom"
+     data-bs-html="true"
+     data-bs-title="{{ cost_tooltip }}"
+     style="position: absolute;
+            bottom: 0px;
+            width: 100%;
+            height: 5px;
+            outline: solid 1px #666;
+            z-index: 1;
+            border-radius: 3px;
+            overflow: hidden">
+     <div id="cost-bar-inner"
```

Comment:
Would using the meter html element be helpful here to simplify some of the logic / closer semantically to what we are trying to achieve?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter

Reply:
Wasn't aware of this. That could work if the color order is reversed with CSS so low = green, high = red.